torch/_higher_order_ops/triton_kernel_wrap.py

```diff
@@ -76,25 +76,18 @@ def __init__(self):
 triton_kernel_wrapper_functional = TritonKernelWrapperFunctional()
 
 
-def grid_fn_code(name, configs, grids):
-    assert len(grids) == len(configs)
-    fn_name = f"grid_wrapper_for_{name}"
-    grid_fn_str = f"def {fn_name}(meta):"
-    for grid, config in zip(grids, configs):
-        guards = [f"meta['{name}'] == {val}" for name, val in config.kwargs.items()]
-        guards = " and ".join(guards)
-        grid_fn_str += f"\n\tif {guards}: return {grid}"
-    return fn_name, grid_fn_str
-
-
 @triton_kernel_wrapper_mutation.py_impl(DispatchKey.CompositeExplicitAutograd)
 def triton_kernel_wrapper_mutation_dense(*, kernel_idx, grid, kwargs):
+    from torch._inductor.codegen.wrapper import user_defined_kernel_grid_fn_code
```

Comment:
I missed some PRs: previously, we were evaluating the grid at Dynamo time. Did that change?

Reply:
We are still doing the same, this is just for emitting the python code for multi option grid function.

From above
> This PR moves the grid function codegen to wrapper so that we can use IndentBuffers as opposed to manually adding tabs for indentation.

Reply:
gotcha, thanks for clarifying